ex03.ex

```diff
@@ -77,7 +84,9 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([], _value),   do: false
```

Comment:
## -1

It's a nice approach, but I think it is clearer using pattern matching to enumerate the options:

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~